packages/SidePanel/src/components/Header/Header.js

```diff
@@ -31,24 +34,27 @@ function darkBackgroundProps(kind) {
 const Header = React.forwardRef((props, ref) => {
   const {
     hasCloseButton,
-    kind,
     isCompact,
+    kind,
+    level,
     // injected by Dialog.js
     onClose,
     ...moreProps
   } = props;
 
   return (
     <div data-pka-anchor="sidepanel.header" ref={ref} css={headerCSS} kind={kind} isCompact={isCompact} {...moreProps}>
-      <div tabIndex="-1">{props.children}</div>
+      <Heading tabIndex="-1" level={level}>
```

Comment:
As per JP's comment, we should include the appropriate `displayLevel` prop here.

Reference: https://github.com/acl-services/paprika/pull/304#issuecomment-558300711